src/App/Pages/Vault/GroupingsPage/GroupingsPageListItem.cs

```diff
@@ -115,5 +116,44 @@ public string Icon
                 return _icon;
             }
         }
-    }
+
+        public string AutomationId
+        {
+            get
+            {
+                if (Type != null)
+                {
+                    switch (Type.Value)
+                    {
+                        case CipherType.Login:
+                            return "LoginFilter";
+                        case CipherType.SecureNote:
+                            return "SecureNoteFilter";
+                        case CipherType.Card:
+                            return "CardFilter";
+                        case CipherType.Identity:
+                            return "IdentityFilter";
+                    }
+                }
+                else if (IsTrash)
+                {
+                    return "TrashFilter";
+                }
+                else if (Folder != null)
+                {
+                    return "FolderFilter";
+                }
+                else if (Collection != null)
+                {
+                    return "CollectionFilter";
+                }
+                else if (IsTotpCode)
+                {
+                    return "TOTPListItem";
+                }
```

Comment:
⛏️  Given that you are returning on each part, then the `else` keyword is not needed, you can just use `if (...)` directly cause the path will finish on each `return ...`